### PR TITLE
Implement full-stack-lab image.

### PR DESCRIPTION
**Pull request by csadorf**
Combines the lab and full-stack images.

### CODE REVIEW

File: docker-bake.hcl [hunk 1]
```diff
@@ -33,7 +33,7 @@ function "tags" {
 }
 
 group "default" {
-  targets = ["base", "full-stack", "lab"]
+  targets = ["base", "full-stack", "lab", "full-stack-lab"]
```
Comment (yakutovicha) [inline docker-bake.hcl:36]: I would very much prefer to name **this** image `full-stack`

File: stack/full-stack-lab/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM full-stack as base
+
+FROM  lab
+
+USER root
+
+COPY --from=base /opt/config-quick-setup.yaml /opt/
+COPY --from=base "${CONDA_DIR}/envs/aiida-core-services" "${CONDA_DIR}/envs/aiida-core-services"
```
Comment (yakutovicha) [inline stack/full-stack-lab/Dockerfile:9]: That can only work if we manage to install RMQ through conda for the arm64 architecture.

**Review by yakutovicha — COMMENTED** [review]
More or less the same questions as for #279.

File: docker-bake.hcl (hunk 1 above)
Comment (csadorf) [reply docker-bake.hcl:36]: Ok, unless you object I will adopt the following image names:

- base: only contains AiiDA
- base-with-services: like base + services (installed and started)
- lab: base + AiiDAlab components
- full-stack: AiiDAlab + services

File: stack/full-stack-lab/Dockerfile (hunk 2 above)
Comment (csadorf) [reply stack/full-stack-lab/Dockerfile:9]: That appears to be the case. With https://github.com/aiidalab/aiidalab-docker-stack/pull/279/commits/02449081481cf050ef1e2c1b9bac7b5c6f7b0874 we build (albeit not necessarily test) all image for both architectures.

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**